docs/release_notes.md

```diff
@@ -33,6 +36,20 @@
 - Fixed import of the constraint weight arrays on sparse arrays. *AdonisFX-1155*
 - Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on initialisation. *AdonisFX-1165*
 - Fixed paint flood operation not normalising the weights properly. *AdonisFX-1146*
+- Fixed a bug that occurred when data from the target mesh was missing. *AdonisFX-1183*
+- Fixed a bug that allowed to remove target geometries while painting Slide On Geometry constraints. *AdonisFX-1192*
+- Fixed a bug that allowed to open the AdonisFX Paint Tool for the final mesh of an AdnSkinMerge deformer. *AdonisFX-1198*
+- Fixed AdonisFX Paint Tool initialization of simulated nodes with intermediate nodes. *AdonisFX-1184*
+
+### Improvements
+
+- Allow source nodes of any type in the *Sensors Connection Editor*. *AdonisFX-1189*
+- Prevent the use of simulated muscles as geometry targets. *AdonisFX-1200*
+
+### Known Limitations
+
+- The *Max Sliding Distance* parameter in AdnSkin and AdnSimshape is represented in scene units. The higher this value is, the more units in the space the sliding constraint allows a vertex to slide on. If the polygons of the target geometry to slide on are very little compared to that value, then the sliding constraint will be time and memory consuming. *AdonisFX-997*
```

Comment:
997 is done. Please remove this item

Reply:
Removed.

Reply:
Is 997 done? We added a dialog but the issue is still there. I remember that we already discuss about having to expose this one.

Reply:
It is true that even though we have integrated the warning dialog, if the user continues with a value that is too high for a mesh, Maya will freeze due to the initialization of the slide constraints. However, I don't know if we should expose this as a limitation since this is how slide constraints work.

That said, should we expose it as a limitation or somewhere else?

Reply:
I don't think we should expose that as a limitatation. If we use for example more subdivisions for a mesh Maya will warn the user. However, if skipped, depending on the case, maya will still freeze. I think with exposing the warning we are informing the user correctly and don't need to expose it as a limitation